ui/admin/app/components/form/target/ssh/index.hbs

```diff
@@ -0,0 +1,7 @@
+{{component
+  (concat 'form/target/details')
```

Comment:
Could you explain what this line is doing exactly please? I am not entirely sure why `concat` is being called here.

Reply:
this has been updated to invoke the component directly!